docs/Legal/Customer Agreement.mdx

```diff
@@ -36,6 +36,22 @@ Si decide utilizar nuestros servicios, acepta pagar las tarifas asociadas con el
 y proporcionarnos un método de pago válido. Si su método de pago falla repetidamente, nos reservamos el derecho de suspender o cancelar su acceso al Servicio, aunque primero se recopilará y enviará una copia de los datos almacenados
 a usted.
 
+### Soporte Técnico
```

Comment:
To make it easy to find from the search engine, I recommend including Service Level Agreement or  "Acuerdo de Nivel de Servicio"somewhere in this section.

Reply:
done